varnish:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_varnish:stable_2024-06-13_18:51:42.json varnish:stable] execution failed: exit status 1
Output: 2024-06-13T18:51:43Z	INFO	Need to update DB
2024-06-13T18:51:43Z	INFO	Downloading DB...	repository="ghcr.io/aquasecurity/trivy-db:2"
23.89 MiB / 48.04 MiB [----------------------------------------------------------->___________________________________________________________] 49.72% ? p/s ?48.04 MiB / 48.04 MiB [--------------------------------------------------------------------------------------------------------------------->] 100.00% ? p/s ?48.04 MiB / 48.04 MiB [--------------------------------------------------------------------------------------------------------------------->] 100.00% ? p/s ?48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 40.24 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 40.24 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 40.24 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 37.64 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 37.64 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 37.64 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 35.21 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 35.21 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-------------------------------------------------------------------------------------------------------->] 100.00% 35.21 MiB p/s ETA 0s48.04 MiB / 48.04 MiB [-----------------------------------------------------------------------------------------------------------] 100.00% 20.08 MiB p/s 2.6s2024-06-13T18:51:46Z	INFO	Vulnerability scanning is enabled
2024-06-13T18:51:46Z	FATAL	Fatal error	image scan error: scan error: scan failed: failed analysis: analyze error: pipeline error: failed to analyze layer (sha256:07701046b65e19aac0a05869083bf6f326998092bdb2106c7305fc3a20c4d3d0): unable to get uncompressed layer sha256:07701046b65e19aac0a05869083bf6f326998092bdb2106c7305fc3a20c4d3d0: failed to get the layer (sha256:07701046b65e19aac0a05869083bf6f326998092bdb2106c7305fc3a20c4d3d0): unable to populate: unable to open: failed to copy the image: image "varnish:stable": not found
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
varnish:old ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
varnish:fresh ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
varnish:7.5.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
varnish:7.5 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
archlinux:multilib-devel-20240101.0.204074 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
archlinux:multilib-devel ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
archlinux:base-devel-20240101.0.204074 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
archlinux:base-devel ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
archlinux:base-20240101.0.204074 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
friendica:stable-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
friendica:stable-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
friendica:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
friendica:fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
friendica:dev-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
caddy:builder-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
caddy:builder ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
caddy:alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
caddy:2.8.4-builder-alpine ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
caddy:2.8.4-builder ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
monica:5.0.0-beta.4-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
monica:5.0.0-beta.4-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
monica:5.0.0-beta-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
monica:5.0.0-beta-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
monica:5.0-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
phpmyadmin:fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
phpmyadmin:apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
phpmyadmin:5.2.1-fpm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
phpmyadmin:5.2.1-apache ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
phpmyadmin:5.2.1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
hitch:1.8.0-1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
hitch:1.8.0 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
hitch:1.8 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
hitch:1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
hitch:1.7.3-1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
almalinux:minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
almalinux:9.4-minimal-20240530 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
almalinux:9.4-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
almalinux:9.4-20240530 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
almalinux:9.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_almalinux:9.4_2024-06-13_19:11:23.json almalinux:9.4] execution failed: exit status 1
Output: 2024-06-13T19:11:24Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:11:24Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "almalinux:9.4" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (almalinux:9.4): Error response from daemon: No such image: almalinux:9.4
	* containerd error: image not found in containerd store: almalinux:9.4
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/almalinux/manifests/9.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_almalinux:9.4_2024-06-13_19:11:23.json almalinux:9.4] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image almalinux:9.4: failed to pull image almalinux:9.4: GET https://index.docker.io/v2/library/almalinux/manifests/9.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
dart:stable-sdk ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_dart:stable-sdk_2024-06-13_19:11:40.json dart:stable-sdk] execution failed: exit status 1
Output: 2024-06-13T19:11:41Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:11:41Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "dart:stable-sdk" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (dart:stable-sdk): Error response from daemon: No such image: dart:stable-sdk
	* containerd error: image not found in containerd store: dart:stable-sdk
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/dart/manifests/stable-sdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_dart:stable-sdk_2024-06-13_19:11:40.json dart:stable-sdk] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image dart:stable-sdk: failed to pull image dart:stable-sdk: GET https://index.docker.io/v2/library/dart/manifests/stable-sdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
dart:stable ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_dart:stable_2024-06-13_19:12:24.json dart:stable] execution failed: exit status 1
Output: 2024-06-13T19:12:25Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:12:25Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "dart:stable" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (dart:stable): Error response from daemon: No such image: dart:stable
	* containerd error: image not found in containerd store: dart:stable
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/dart/manifests/stable: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_dart:stable_2024-06-13_19:12:24.json dart:stable] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image dart:stable: failed to pull image dart:stable: GET https://index.docker.io/v2/library/dart/manifests/stable: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
dart:sdk ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_dart:sdk_2024-06-13_19:12:57.json dart:sdk] execution failed: exit status 1
Output: 2024-06-13T19:12:57Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:12:58Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "dart:sdk" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (dart:sdk): Error response from daemon: No such image: dart:sdk
	* containerd error: image not found in containerd store: dart:sdk
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/dart/manifests/sdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_dart:sdk_2024-06-13_19:12:57.json dart:sdk] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image dart:sdk: failed to pull image dart:sdk: GET https://index.docker.io/v2/library/dart/manifests/sdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
dart:beta-sdk ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_dart:beta-sdk_2024-06-13_19:13:33.json dart:beta-sdk] execution failed: exit status 1
Output: 2024-06-13T19:13:34Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:13:34Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "dart:beta-sdk" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (dart:beta-sdk): Error response from daemon: No such image: dart:beta-sdk
	* containerd error: image not found in containerd store: dart:beta-sdk
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/dart/manifests/beta-sdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_dart:beta-sdk_2024-06-13_19:13:33.json dart:beta-sdk] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image dart:beta-sdk: failed to pull image dart:beta-sdk: GET https://index.docker.io/v2/library/dart/manifests/beta-sdk: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
dart:beta ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_dart:beta_2024-06-13_19:14:18.json dart:beta] execution failed: exit status 1
Output: 2024-06-13T19:14:19Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:14:20Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "dart:beta" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (dart:beta): Error response from daemon: No such image: dart:beta
	* containerd error: image not found in containerd store: dart:beta
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/dart/manifests/beta: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_dart:beta_2024-06-13_19:14:18.json dart:beta] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image dart:beta: failed to pull image dart:beta: GET https://index.docker.io/v2/library/dart/manifests/beta: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-temurin:8u412-b08-jre-ubi9-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_eclipse-temurin:8u412-b08-jre-ubi9-minimal_2024-06-13_19:14:50.json eclipse-temurin:8u412-b08-jre-ubi9-minimal] execution failed: exit status 1
Output: 2024-06-13T19:14:51Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:14:52Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "eclipse-temurin:8u412-b08-jre-ubi9-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (eclipse-temurin:8u412-b08-jre-ubi9-minimal): Error response from daemon: No such image: eclipse-temurin:8u412-b08-jre-ubi9-minimal
	* containerd error: image not found in containerd store: eclipse-temurin:8u412-b08-jre-ubi9-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8u412-b08-jre-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_eclipse-temurin:8u412-b08-jre-ubi9-minimal_2024-06-13_19:14:50.json eclipse-temurin:8u412-b08-jre-ubi9-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image eclipse-temurin:8u412-b08-jre-ubi9-minimal: failed to pull image eclipse-temurin:8u412-b08-jre-ubi9-minimal: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8u412-b08-jre-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-temurin:8u412-b08-jdk-ubi9-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_eclipse-temurin:8u412-b08-jdk-ubi9-minimal_2024-06-13_19:15:13.json eclipse-temurin:8u412-b08-jdk-ubi9-minimal] execution failed: exit status 1
Output: 2024-06-13T19:15:14Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:15:14Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "eclipse-temurin:8u412-b08-jdk-ubi9-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (eclipse-temurin:8u412-b08-jdk-ubi9-minimal): Error response from daemon: No such image: eclipse-temurin:8u412-b08-jdk-ubi9-minimal
	* containerd error: image not found in containerd store: eclipse-temurin:8u412-b08-jdk-ubi9-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8u412-b08-jdk-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_eclipse-temurin:8u412-b08-jdk-ubi9-minimal_2024-06-13_19:15:13.json eclipse-temurin:8u412-b08-jdk-ubi9-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image eclipse-temurin:8u412-b08-jdk-ubi9-minimal: failed to pull image eclipse-temurin:8u412-b08-jdk-ubi9-minimal: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8u412-b08-jdk-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
eclipse-temurin:8-ubi9-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_eclipse-temurin:8-ubi9-minimal_2024-06-13_19:15:39.json eclipse-temurin:8-ubi9-minimal] execution failed: exit status 1
Output: 2024-06-13T19:15:40Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:15:40Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "eclipse-temurin:8-ubi9-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (eclipse-temurin:8-ubi9-minimal): Error response from daemon: No such image: eclipse-temurin:8-ubi9-minimal
	* containerd error: image not found in containerd store: eclipse-temurin:8-ubi9-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_eclipse-temurin:8-ubi9-minimal_2024-06-13_19:15:39.json eclipse-temurin:8-ubi9-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image eclipse-temurin:8-ubi9-minimal: failed to pull image eclipse-temurin:8-ubi9-minimal: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
eclipse-temurin:8-jre-ubi9-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_eclipse-temurin:8-jre-ubi9-minimal_2024-06-13_19:16:02.json eclipse-temurin:8-jre-ubi9-minimal] execution failed: exit status 1
Output: 2024-06-13T19:16:03Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:16:04Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "eclipse-temurin:8-jre-ubi9-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (eclipse-temurin:8-jre-ubi9-minimal): Error response from daemon: No such image: eclipse-temurin:8-jre-ubi9-minimal
	* containerd error: image not found in containerd store: eclipse-temurin:8-jre-ubi9-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8-jre-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_eclipse-temurin:8-jre-ubi9-minimal_2024-06-13_19:16:02.json eclipse-temurin:8-jre-ubi9-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image eclipse-temurin:8-jre-ubi9-minimal: failed to pull image eclipse-temurin:8-jre-ubi9-minimal: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8-jre-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
eclipse-temurin:8-jdk-ubi9-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_eclipse-temurin:8-jdk-ubi9-minimal_2024-06-13_19:16:24.json eclipse-temurin:8-jdk-ubi9-minimal] execution failed: exit status 1
Output: 2024-06-13T19:16:25Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:16:25Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "eclipse-temurin:8-jdk-ubi9-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (eclipse-temurin:8-jdk-ubi9-minimal): Error response from daemon: No such image: eclipse-temurin:8-jdk-ubi9-minimal
	* containerd error: image not found in containerd store: eclipse-temurin:8-jdk-ubi9-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8-jdk-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_eclipse-temurin:8-jdk-ubi9-minimal_2024-06-13_19:16:24.json eclipse-temurin:8-jdk-ubi9-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image eclipse-temurin:8-jdk-ubi9-minimal: failed to pull image eclipse-temurin:8-jdk-ubi9-minimal: GET https://index.docker.io/v2/library/eclipse-temurin/manifests/8-jdk-ubi9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
ibm-semeru-runtimes:open-8u412-b08-jre-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_ibm-semeru-runtimes:open-8u412-b08-jre-jammy_2024-06-13_19:16:47.json ibm-semeru-runtimes:open-8u412-b08-jre-jammy] execution failed: exit status 1
Output: 2024-06-13T19:16:48Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:16:49Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "ibm-semeru-runtimes:open-8u412-b08-jre-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (ibm-semeru-runtimes:open-8u412-b08-jre-jammy): Error response from daemon: No such image: ibm-semeru-runtimes:open-8u412-b08-jre-jammy
	* containerd error: image not found in containerd store: ibm-semeru-runtimes:open-8u412-b08-jre-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jre-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_ibm-semeru-runtimes:open-8u412-b08-jre-jammy_2024-06-13_19:16:47.json ibm-semeru-runtimes:open-8u412-b08-jre-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image ibm-semeru-runtimes:open-8u412-b08-jre-jammy: failed to pull image ibm-semeru-runtimes:open-8u412-b08-jre-jammy: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jre-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
ibm-semeru-runtimes:open-8u412-b08-jre-focal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_ibm-semeru-runtimes:open-8u412-b08-jre-focal_2024-06-13_19:17:08.json ibm-semeru-runtimes:open-8u412-b08-jre-focal] execution failed: exit status 1
Output: 2024-06-13T19:17:09Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:17:09Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "ibm-semeru-runtimes:open-8u412-b08-jre-focal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (ibm-semeru-runtimes:open-8u412-b08-jre-focal): Error response from daemon: No such image: ibm-semeru-runtimes:open-8u412-b08-jre-focal
	* containerd error: image not found in containerd store: ibm-semeru-runtimes:open-8u412-b08-jre-focal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jre-focal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_ibm-semeru-runtimes:open-8u412-b08-jre-focal_2024-06-13_19:17:08.json ibm-semeru-runtimes:open-8u412-b08-jre-focal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image ibm-semeru-runtimes:open-8u412-b08-jre-focal: failed to pull image ibm-semeru-runtimes:open-8u412-b08-jre-focal: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jre-focal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
ibm-semeru-runtimes:open-8u412-b08-jre ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_ibm-semeru-runtimes:open-8u412-b08-jre_2024-06-13_19:17:28.json ibm-semeru-runtimes:open-8u412-b08-jre] execution failed: exit status 1
Output: 2024-06-13T19:17:29Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:17:29Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "ibm-semeru-runtimes:open-8u412-b08-jre" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (ibm-semeru-runtimes:open-8u412-b08-jre): Error response from daemon: No such image: ibm-semeru-runtimes:open-8u412-b08-jre
	* containerd error: image not found in containerd store: ibm-semeru-runtimes:open-8u412-b08-jre
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jre: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_ibm-semeru-runtimes:open-8u412-b08-jre_2024-06-13_19:17:28.json ibm-semeru-runtimes:open-8u412-b08-jre] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image ibm-semeru-runtimes:open-8u412-b08-jre: failed to pull image ibm-semeru-runtimes:open-8u412-b08-jre: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jre: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
ibm-semeru-runtimes:open-8u412-b08-jdk-jammy ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_ibm-semeru-runtimes:open-8u412-b08-jdk-jammy_2024-06-13_19:17:47.json ibm-semeru-runtimes:open-8u412-b08-jdk-jammy] execution failed: exit status 1
Output: 2024-06-13T19:17:48Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:17:49Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "ibm-semeru-runtimes:open-8u412-b08-jdk-jammy" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (ibm-semeru-runtimes:open-8u412-b08-jdk-jammy): Error response from daemon: No such image: ibm-semeru-runtimes:open-8u412-b08-jdk-jammy
	* containerd error: image not found in containerd store: ibm-semeru-runtimes:open-8u412-b08-jdk-jammy
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jdk-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_ibm-semeru-runtimes:open-8u412-b08-jdk-jammy_2024-06-13_19:17:47.json ibm-semeru-runtimes:open-8u412-b08-jdk-jammy] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image ibm-semeru-runtimes:open-8u412-b08-jdk-jammy: failed to pull image ibm-semeru-runtimes:open-8u412-b08-jdk-jammy: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jdk-jammy: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
ibm-semeru-runtimes:open-8u412-b08-jdk-focal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_ibm-semeru-runtimes:open-8u412-b08-jdk-focal_2024-06-13_19:18:14.json ibm-semeru-runtimes:open-8u412-b08-jdk-focal] execution failed: exit status 1
Output: 2024-06-13T19:18:14Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:18:15Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "ibm-semeru-runtimes:open-8u412-b08-jdk-focal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (ibm-semeru-runtimes:open-8u412-b08-jdk-focal): Error response from daemon: No such image: ibm-semeru-runtimes:open-8u412-b08-jdk-focal
	* containerd error: image not found in containerd store: ibm-semeru-runtimes:open-8u412-b08-jdk-focal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jdk-focal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_ibm-semeru-runtimes:open-8u412-b08-jdk-focal_2024-06-13_19:18:14.json ibm-semeru-runtimes:open-8u412-b08-jdk-focal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image ibm-semeru-runtimes:open-8u412-b08-jdk-focal: failed to pull image ibm-semeru-runtimes:open-8u412-b08-jdk-focal: GET https://index.docker.io/v2/library/ibm-semeru-runtimes/manifests/open-8u412-b08-jdk-focal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil>
rockylinux:9.3.20231119-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_rockylinux:9.3.20231119-minimal_2024-06-13_19:18:40.json rockylinux:9.3.20231119-minimal] execution failed: exit status 1
Output: 2024-06-13T19:18:41Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:18:41Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "rockylinux:9.3.20231119-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (rockylinux:9.3.20231119-minimal): Error response from daemon: No such image: rockylinux:9.3.20231119-minimal
	* containerd error: image not found in containerd store: rockylinux:9.3.20231119-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3.20231119-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_rockylinux:9.3.20231119-minimal_2024-06-13_19:18:40.json rockylinux:9.3.20231119-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image rockylinux:9.3.20231119-minimal: failed to pull image rockylinux:9.3.20231119-minimal: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3.20231119-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rockylinux:9.3.20231119 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_rockylinux:9.3.20231119_2024-06-13_19:18:56.json rockylinux:9.3.20231119] execution failed: exit status 1
Output: 2024-06-13T19:18:57Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:18:58Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "rockylinux:9.3.20231119" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (rockylinux:9.3.20231119): Error response from daemon: No such image: rockylinux:9.3.20231119
	* containerd error: image not found in containerd store: rockylinux:9.3.20231119
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3.20231119: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_rockylinux:9.3.20231119_2024-06-13_19:18:56.json rockylinux:9.3.20231119] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image rockylinux:9.3.20231119: failed to pull image rockylinux:9.3.20231119: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3.20231119: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rockylinux:9.3-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_rockylinux:9.3-minimal_2024-06-13_19:19:15.json rockylinux:9.3-minimal] execution failed: exit status 1
Output: 2024-06-13T19:19:16Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:19:17Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "rockylinux:9.3-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (rockylinux:9.3-minimal): Error response from daemon: No such image: rockylinux:9.3-minimal
	* containerd error: image not found in containerd store: rockylinux:9.3-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_rockylinux:9.3-minimal_2024-06-13_19:19:15.json rockylinux:9.3-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image rockylinux:9.3-minimal: failed to pull image rockylinux:9.3-minimal: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rockylinux:9.3 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_rockylinux:9.3_2024-06-13_19:19:33.json rockylinux:9.3] execution failed: exit status 1
Output: 2024-06-13T19:19:34Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:19:35Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "rockylinux:9.3" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (rockylinux:9.3): Error response from daemon: No such image: rockylinux:9.3
	* containerd error: image not found in containerd store: rockylinux:9.3
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_rockylinux:9.3_2024-06-13_19:19:33.json rockylinux:9.3] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image rockylinux:9.3: failed to pull image rockylinux:9.3: GET https://index.docker.io/v2/library/rockylinux/manifests/9.3: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
rockylinux:9-minimal ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_rockylinux:9-minimal_2024-06-13_19:19:51.json rockylinux:9-minimal] execution failed: exit status 1
Output: 2024-06-13T19:19:52Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:19:53Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "rockylinux:9-minimal" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (rockylinux:9-minimal): Error response from daemon: No such image: rockylinux:9-minimal
	* containerd error: image not found in containerd store: rockylinux:9-minimal
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/rockylinux/manifests/9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_rockylinux:9-minimal_2024-06-13_19:19:51.json rockylinux:9-minimal] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image rockylinux:9-minimal: failed to pull image rockylinux:9-minimal: GET https://index.docker.io/v2/library/rockylinux/manifests/9-minimal: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
api-firewall:0.7.3 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_api-firewall:0.7.3_2024-06-13_19:20:10.json api-firewall:0.7.3] execution failed: exit status 1
Output: 2024-06-13T19:20:11Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:20:12Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "api-firewall:0.7.3" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (api-firewall:0.7.3): Error response from daemon: No such image: api-firewall:0.7.3
	* containerd error: image not found in containerd store: api-firewall:0.7.3
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/api-firewall/manifests/0.7.3: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> <nil>
api-firewall:0.7.2 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_api-firewall:0.7.2_2024-06-13_19:20:20.json api-firewall:0.7.2] execution failed: exit status 1
Output: 2024-06-13T19:20:21Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:20:22Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "api-firewall:0.7.2" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (api-firewall:0.7.2): Error response from daemon: No such image: api-firewall:0.7.2
	* containerd error: image not found in containerd store: api-firewall:0.7.2
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/api-firewall/manifests/0.7.2: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_api-firewall:0.7.2_2024-06-13_19:20:20.json api-firewall:0.7.2] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image api-firewall:0.7.2: failed to pull image api-firewall:0.7.2: GET https://index.docker.io/v2/library/api-firewall/manifests/0.7.2: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
api-firewall:0.7.1 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_api-firewall:0.7.1_2024-06-13_19:20:34.json api-firewall:0.7.1] execution failed: exit status 1
Output: 2024-06-13T19:20:34Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:20:35Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "api-firewall:0.7.1" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (api-firewall:0.7.1): Error response from daemon: No such image: api-firewall:0.7.1
	* containerd error: image not found in containerd store: api-firewall:0.7.1
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/api-firewall/manifests/0.7.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_api-firewall:0.7.1_2024-06-13_19:20:34.json api-firewall:0.7.1] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image api-firewall:0.7.1: failed to pull image api-firewall:0.7.1: GET https://index.docker.io/v2/library/api-firewall/manifests/0.7.1: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> <nil>
api-firewall:0.7.0 ----------- Error pulling image api-firewall:0.7.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
api-firewall:0.6.17 ----------- Error pulling image api-firewall:0.6.17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.7.0 ----------- Error pulling image emqx:5.7.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.7 ----------- Error pulling image emqx:5.7: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5 ----------- Error pulling image emqx:5: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.6.1 ----------- Error pulling image emqx:5.6.1: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
emqx:5.6 ----------- Error pulling image emqx:5.6: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:bookworm ----------- Error pulling image satosa:bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4.0-bookworm ----------- Error pulling image satosa:8.4.0-bookworm: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4.0 ----------- Error pulling image satosa:8.4.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
satosa:8.4-bookworm ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_satosa:8.4-bookworm_2024-06-13_19:21:00.json satosa:8.4-bookworm] execution failed: exit status 1
Output: 2024-06-13T19:21:01Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:21:02Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "satosa:8.4-bookworm" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (satosa:8.4-bookworm): Error response from daemon: No such image: satosa:8.4-bookworm
	* containerd error: image not found in containerd store: satosa:8.4-bookworm
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/satosa/manifests/8.4-bookworm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_satosa:8.4-bookworm_2024-06-13_19:21:00.json satosa:8.4-bookworm] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image satosa:8.4-bookworm: failed to pull image satosa:8.4-bookworm: GET https://index.docker.io/v2/library/satosa/manifests/8.4-bookworm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new10/Grype_scan_satosa:8.4-bookworm_2024-06-13_19:21:00.json satosa:8.4-bookworm] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'satosa:8.4-bookworm':
  - docker: unable to inspect existing image: Error response from daemon: No such image: satosa:8.4-bookworm
  - podman: podman not available: no host address
  - containerd: pull failed: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/library/satosa/manifests/sha256:9e800fdab733753e6519b6fef827beb7b2086118b79c05414558cd3fbb017d05: 429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - oci-registry: failed to get image descriptor from registry: GET https://index.docker.io/v2/library/satosa/manifests/8.4-bookworm: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test satosa:8.4-bookworm --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new10/Snyk_scan_satosa:8.4-bookworm_2024-06-13_19:21:00.json] execution failed: exit status 2
Output: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit

satosa:8.4 ----------- <nil> 
 NEW ERROR: [---ERROR WITH SCANNER---Trivy] --->> Command [/usr/bin/trivy image -f sarif --scanners vuln -o /root/docker-vulnerability-analyzer/out/official/new10/Trivy_scan_satosa:8.4_2024-06-13_19:21:10.json satosa:8.4] execution failed: exit status 1
Output: 2024-06-13T19:21:11Z	INFO	Vulnerability scanning is enabled
2024-06-13T19:21:12Z	FATAL	Fatal error	image scan error: scan error: unable to initialize a scanner: unable to initialize an image scanner: unable to find the specified image "satosa:8.4" in ["docker" "containerd" "podman" "remote"]: 4 errors occurred:
	* docker error: unable to inspect the image (satosa:8.4): Error response from daemon: No such image: satosa:8.4
	* containerd error: image not found in containerd store: satosa:8.4
	* podman error: unable to initialize Podman client: no podman socket found: stat /run/user/0/podman/podman.sock: no such file or directory
	* remote error: GET https://index.docker.io/v2/library/satosa/manifests/8.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit


 
 NEW ERROR: [---ERROR WITH SCANNER---DockerScout] --->> Command [/usr/bin/docker scout cves --format sarif --epss --details --locations --output /root/docker-vulnerability-analyzer/out/official/new10/DockerScout_scan_satosa:8.4_2024-06-13_19:21:10.json satosa:8.4] execution failed: exit status 1
Output:     ...Pulling
    ✗ Pull failed
[31mERROR  [0m Status: failed to get image satosa:8.4: failed to pull image satosa:8.4: GET https://index.docker.io/v2/library/satosa/manifests/8.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit, Code: 1 
 
 NEW ERROR: [---ERROR WITH SCANNER---Grype] --->> Command [/usr/local/bin/grype -o sarif --file /root/docker-vulnerability-analyzer/out/official/new10/Grype_scan_satosa:8.4_2024-06-13_19:21:10.json satosa:8.4] execution failed: exit status 1
Output: [31mfailed to catalog: errors occurred attempting to resolve 'satosa:8.4':
  - docker: unable to inspect existing image: Error response from daemon: No such image: satosa:8.4
  - podman: podman not available: no host address
  - containerd: pull failed: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/library/satosa/manifests/sha256:9e800fdab733753e6519b6fef827beb7b2086118b79c05414558cd3fbb017d05: 429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - oci-registry: failed to get image descriptor from registry: GET https://index.docker.io/v2/library/satosa/manifests/8.4: TOOMANYREQUESTS: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
  - additionally, the following providers failed with file does not exist: docker-archive, oci-archive, oci-dir, singularity, oci-dir, local-file, local-directory[0m
 
 NEW ERROR: [---ERROR WITH SCANNER---Snyk] --->> Command [/usr/local/bin/snyk container test satosa:8.4 --sarif-file-output=/root/docker-vulnerability-analyzer/out/official/new10/Snyk_scan_satosa:8.4_2024-06-13_19:21:10.json] execution failed: exit status 2
Output: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit

unit:php8.3 ----------- Error pulling image unit:php8.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8.2 ----------- Error pulling image unit:php8.2: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php8 ----------- Error pulling image unit:php8: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:php ----------- Error pulling image unit:php: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
unit:1.32.1-php8.3 ----------- Error pulling image unit:1.32.1-php8.3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:scala ----------- Error pulling image spark:scala: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:r ----------- Error pulling image spark:r: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:python3-java17 ----------- Error pulling image spark:python3-java17: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:python3 ----------- Error pulling image spark:python3: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
spark:3.5.1-scala2.12-java17-ubuntu ----------- Error pulling image spark:3.5.1-scala2.12-java17-ubuntu: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28.0 ----------- Error pulling image liquibase:4.28.0: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28 ----------- Error pulling image liquibase:4.28: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:alpine ----------- Error pulling image liquibase:alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28.0-alpine ----------- Error pulling image liquibase:4.28.0-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
liquibase:4.28-alpine ----------- Error pulling image liquibase:4.28-alpine: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
